docs/admin/admission-controllers.md

```diff
@@ -468,6 +468,18 @@ metadata:
   name: namespace3
 ```
 
+#### Internal Behavior
+This admission controller has the following behavior:
+  1. If the namespace has an annotation with a key `scheduler.kubernetes.io/nodeSelector`, use its value as the
+     namespace node selector.
+  2. If the namespace lacks such an annotation, use the `clusterDefaultNodeSelector` defined in the plugin
```

Comment:
I believe the convention is to use `1.` for all items in an order list.

Reply:
For consistency's sake, say "the PodNodeSelector configuration file" instead of "the plugin configuration file" because the latter one is ambiguous.